storagetransfer/posix_to_posix_request.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that creates a transfer from a POSIX file system to GCS.
+"""
+
+
+import argparse
+
+# [START storagetransfer_transfer_posix_to_posix]
+from google.cloud import storage_transfer
+
+
+def transfer_between_posix(
+        project_id: str, description: str, source_agent_pool_name: str,
+        sink_agent_pool_name: str, root_directory: str,
+        destination_directory: str, intermediate_bucket: str):
+    """Create a transfer from a POSIX file system to a GCS bucket."""
```

Comment:
This description is the same as the one in `storagetransfer/posix_request.py`. Is it correct?